src/DssSpell.sol

```diff
@@ -39,13 +43,71 @@ contract DssSpellAction is DssAction {
     //
     // uint256 internal constant X_PCT_RATE      = ;
 
-    address immutable internal ESM            = DssExecLib.getChangelogAddress("MCD_ESM");
-    address immutable internal RWA_LIQ_ORACLE = DssExecLib.getChangelogAddress("MIP21_LIQUIDATION_ORACLE");
+    uint256 internal constant FOUR_NINE_PCT_RATE = 1000000001516911765932351183;
+
+    uint256 internal constant MILLION = 10**6;
+    uint256 internal constant RAD     = 10**45;
 
     function actions() public override {
 
-        // ----- Additional ESM authorization fix -----
-        DssExecLib.authorize(RWA_LIQ_ORACLE, ESM);
+        uint256 Art;
+        uint256 rate;
+        uint256 line;
+        uint256 sumLines;
```

Comment:
It would be better to follow existing standards as per the archives 

Here is an example of a blank offboarding template:
https://github.com/makerdao/spells-mainnet/blob/5dc5b14661b2fbaebaaa7a3f2b48cfffb63a669c/archive/2022-10-26-DssSpell/DssSpellCollateral.sol#L113

Reply:
Done.